app/main.py

```
@@ -1,12 +1,42 @@
+    app = FastAPI(
+        title="Calendar Server",
+        description="Calendar Server API",
+        version="1.0.0",
+        docs_url="/docs",
+        redoc_url="/redoc",
+    )
+
+    app.add_exception_handler(DomainException, domain_exception_handler)
+
+    # HTTP 예외 핸들러
+    from fastapi import HTTPException
```

Ruff (0.12.2)

Redefinition of unused `HTTPException` from line 4

Remove definition: `HTTPException`

(F811)